sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1384,7 +1384,9 @@ public partial class BlobSasBuilder
         public void SetPermissions(Azure.Storage.Sas.BlobSasPermissions permissions) { }
         public void SetPermissions(Azure.Storage.Sas.BlobVersionSasPermissions permissions) { }
         public void SetPermissions(Azure.Storage.Sas.SnapshotSasPermissions permissions) { }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public void SetPermissions(string rawPermissions) { }
+        public void SetPermissions(string rawPermissions, bool normalize = false) { }
```

Comment:
Nit - we mostly use the `[EditorBrowsable(Never)]` for hiding overloads with optional params.  We'd typically solve this case with an extra overload.